src/extension.ts

```diff
@@ -235,6 +239,24 @@ function registerStopExecution(context: ExtensionContext) {
     context.subscriptions.push(commands.registerCommand(GaugeVSCodeCommands.StopExecution, () => { cancel(); }));
 }
 
+function registerExecutionStatus(context: ExtensionContext) {
+    let executionStatus = window.createStatusBarItem(StatusBarAlignment.Left, 1);
+    executionStatus.command = GaugeVSCodeCommands.QuickPick;
+    context.subscriptions.push(executionStatus);
+    onExecuted((projectRoot) => {
+        let languageClient = clients.get(Uri.file(projectRoot).fsPath);
+        return languageClient.sendRequest("gauge/executionStatus", {}, new CancellationTokenSource().token).then(
+            (val: any) => {
+                executionStatus.text = val.ScePassed.toString() + `$(check) ` + val.SceFailed.toString() + `$(x) ` + val.SceSkipped.toString() +`$(issue-opened)`;
+                executionStatus.tooltip ="Specs : " + val.SpecsExecuted.toString() + " Executed, " + val.SpecsPassed.toString() + " Passed, " + val.SpecsFailed.toString()+" Failed, " + val.SpecsSkipped.toString() + " Skipped" + "\n" +
+                                        "Scenarios : " + val.SceExecuted.toString() + " Executed, " + val.ScePassed.toString() + " Passed, " + val.SceFailed.toString()+" Failed, " + val.SceSkipped.toString() + " Skipped";
+                executionStatus.show();
+            }
+        );
+    });
+    context.subscriptions.push(commands.registerCommand(GaugeVSCodeCommands.QuickPick, () => { showQuickPickItems(); }));
+}
```

Comment:
name showQuickPickItems is very generic. rename it to something context specific